workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/PurchasedGetAppPage/PurchasedGetAppAccountSelection.tsx

```diff
@@ -0,0 +1,221 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import ClayButton from '@clayui/button';
+import ClayForm, {ClayRadio} from '@clayui/form';
+import ClayLink from '@clayui/link';
+import {useEffect, useState} from 'react';
+
+import emptyPictureIcon from '../../assets/icons/avatar.svg';
+import {Header} from '../../components/Header/Header';
+
+import './PurchasedGetAppPage.scss';
+
+import ClaySticker from '@clayui/sticker';
+import classNames from 'classnames';
+
+import {getChannels, getOrderTypes, postOrder} from '../../utils/api';
+
+type Steps = {
+	page: 'accountCreation' | 'accountSelection' | 'projectCreated';
+};
+
+type PurchasedGetAppAccountSelectionProps = {
+	accounts: Account[];
+	currentUserAccount?: UserAccount;
+	orderInfo?: OrderInfo;
+	setStep: React.Dispatch<Steps>;
+};
+
+const PurchasedGetAppAccountSelection: React.FC<
+	PurchasedGetAppAccountSelectionProps
+> = ({accounts, currentUserAccount, orderInfo, setStep}) => {
+	const [radio, setRadio] = useState<RadioOption>();
+	const [orderType, setOrderType] = useState<OrderType>();
+
+	const [channel, setChannel] = useState<Channel>({
+		channelId: 0,
+		currencyCode: '',
+		externalReferenceCode: '',
+		id: 0,
+		name: '',
+		siteGroupId: 0,
+		type: '',
+	});
+
+	useEffect(() => {
+		(async () => {
+			const channels = await getChannels();
+
+			const channel =
+				channels.find(
+					(channel) => channel.name === 'Marketplace Channel'
+				) || channels[0];
+			setChannel(channel);
+
+			const orderTypes = await getOrderTypes();
+			const projectOrderType = orderTypes.find(
+				({name}) => name['en_US'] === 'Solutions - 30 day trial'
+			);
```

Comment:
I think that will be best to use ERC to find the right order type, let me know if you agree